NEWS.md

```diff
@@ -478,6 +478,17 @@ Deprecated or removed
   * The no-op `transpose` fallback has been deprecated. Consider introducing suitable
     `transpose` methods or calling `permutedims(x, [2,1])` ([#13171], [#17075], [#17374]).
 
+  * The following macros have been deprecated ([#18189]):
```

Comment:
18189 is not the right reference, it should refer to when the actual change was made